### DISCUSSION TITLE
Filament Group rows group button

### DISCUSSION THREAD

**Discussion by RafhaanVilla** [filamentphp/filament#13051] [Help] [answered]
### Package

Table builder

### Package Version

V3.2

### How can we help you?

Is there any way to add a button the the grouped row to update the things only in the group. right now i am using bulk actions to edit the groups as i can select the group, and hidding the checkbox when the status is delivered. but the checkbox doesnt hide from the grouped row. 
<img width="863" alt="Screenshot 2024-06-01 at 1 30 18 PM" src="https://github.com/filamentphp/filament/assets/152064130/12ad8efb-b688-4d1f-b81b-993942dd5a58">

**Comment by leandrocfe** [comment, marked as answer]
I think there isn't an option to do this
